AppCenter/AppCenter/Internals/Util/MSConstants+Internal.h

```diff
@@ -157,3 +157,8 @@ static NSString *const kMSOneCollectorGroupIdSuffix = @"/one";
  * Bit mask for persistence flags.
  */
 static const NSUInteger kMSPersistenceFlagsMask = 0xFF;
+
+/**
+ * Valid application userId for One Collector.
+ */
+static NSString *const kAppUserIdPattern = @"^[cidw]:.*$";
```

Comment:
This pattern is not used, is this on purpose?

Reply:
Yes, it is common code and will be used in https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1229/files#diff-98b550149ebafa1f543459a25aff369eR55

Reply:
We introduce it now to avoid conflicts with the other task, we want to share this PR between the 2 tasks.